### ISSUE TITLE
need help 

### ISSUE THREAD

**Issue by webAtmTech** [wahyukaprawi/learn_flutter_firebase_authentication#1]
Your code is nicely written.  Let me know if you would consider paid consultation, say on Fiverr.   let me know how to reach you on google chat, telegram or whatever.

**Comment by wahyukaprawi** [comment]
Of course, I am open to instagram. You can contact me at whyukaprawi to discuss further. Thank you!
